src/Microsoft.IdentityModel.Tokens/Validation/Validators.Issuer.cs

```diff
@@ -95,10 +91,8 @@ public static partial class Validators
             if (validationParameters.ConfigurationManager != null)
                 configuration = await validationParameters.ConfigurationManager.GetBaseConfigurationAsync(cancellationToken).ConfigureAwait(false);
 
-            // Throw if all possible places to validate against are null or empty
-            if (string.IsNullOrWhiteSpace(validationParameters.ValidIssuer)
-                && validationParameters.ValidIssuers.IsNullOrEmpty()
-                && string.IsNullOrWhiteSpace(configuration?.Issuer))
+            // Return failed IssuerValidationResult if all possible places to validate against are null or empty.
+            if (validationParameters.ValidIssuers.IsNullOrEmpty() && string.IsNullOrWhiteSpace(configuration?.Issuer))
```

Comment:
Similar to Brent's comment on the other PR, now that `ValidIssuers` is an `IList`, we should check `.Count == 0`.

Reply:
Looks like we are still using `IsNullOrEmpty()` on an `IList`.

The comment above needs to be addressed.